EventListener/VoteBlamerListener.php

```diff
@@ -25,10 +27,17 @@
  */
 class VoteBlamerListener implements EventSubscriberInterface
 {
+    protected $securityContext;
```

Comment:
Fixed in https://github.com/FriendsOfSymfony/FOSCommentBundle/commit/5915080d236126ad248945d0ef713ae9dc656020